src/Menu/Menu.js

```diff
@@ -15,83 +15,103 @@ const Menu = React.createClass({
 
   propTypes: {
     /**
-     * If true, the menu will apply transitions when added it
-     * gets added to the DOM. In order for transitions to
-     * work, wrap the menu inside a ReactTransitionGroup.
+     * If true, the menu will apply transitions when it
+     * is added to the DOM. In order for transitions to
+     * work, wrap the menu inside a `ReactTransitionGroup`.
      */
     animated: deprecated(React.PropTypes.bool, 'Instead, use a [Popover](/#/components/popover).'),
 
     /**
-     * If true, the width will automatically be
-     * set according to the items inside the menu
-     * using the proper keyline increment.
+     * If true, the width of the menu will automatically be
+     * set according to the widths of its children,
+     * using proper keyline increments (64px for desktop,
+     * 56px otherwise).
      */
     autoWidth: React.PropTypes.bool,
 
     /**
-     * Children for the Menu. Usually MenuItems.
+     * The content of the menu. This is usually used to pass `MenuItem`
+     * elements.
      */
     children: React.PropTypes.node,
 
     /**
-     * Indicates if the menu should render with compact desktop styles.
+     * If true, the menu item will render with compact desktop styles.
      */
     desktop: React.PropTypes.bool,
 
     /**
-     * Disable the auto focus feature.
+     * If true, the menu will not be auto-focused.
      */
     disableAutoFocus: React.PropTypes.bool,
 
     /**
-     * True if this item should be focused by the keyboard initially.
+     * If true, the menu will be keyboard-focused initially.
      */
     initiallyKeyboardFocused: React.PropTypes.bool,
 
     /**
-     * The style object to use to override underlying list style.
+     * Override the inline-styles of the underlying `List` element.
      */
     listStyle: React.PropTypes.object,
 
     /**
-     * The maxHeight of the menu in pixels. If
-     * specified, the menu will scroll if larger than the maxHeight.
+     * The maximum height of the menu in pixels. If specified,
+     * the menu will be scrollable if it's taller than the provided
+     * height.
      */
     maxHeight: React.PropTypes.number,
 
     /**
-     * If true, the value can be an array and allow the menu to be a multi-select.
+     * If true, `value` must be an array and the menu will support
+     * multiple selections.
      */
     multiple: React.PropTypes.bool,
```

Comment:
Seems like we should just be detecting the type of value, rather than having a prop to enforce it?